lib/ansible/modules/commands/command.py

```diff
@@ -81,6 +81,31 @@
   register: myoutput
 '''
 
+RETURN = '''
+cmd:
+  description: the cmd that was run on the remote machine
+  returned: always
+  type: list
+  sample:
+  - echo
+  - hello
+delta:
+  description: cmd end time - cmd start time
+  returned: always
+  type: string
+  sample: '0:00:00.001529'
```

Comment:
Please don't add quotes when they're not necessary.
People may have te impression quoting is required here, but it's not.